src/js/controller/model.js

```diff
@@ -264,11 +268,11 @@ define([
         this.loadVideo = function() {
             this.mediaController.trigger(events.JWPLAYER_MEDIA_PLAY_ATTEMPT);
             var idx = this.get('item');
-            this.getVideo().load(this.get('playlist')[idx]);
+            _provider.load(this.get('playlist')[idx]);
         };
 
         this.playVideo = function() {
-            this.getVideo().play();
+            _provider.play();
```

Comment:
Not valid at this point because _provider is being used more than getVideo(). I just went with the one we use more, preferring to use internal vars over public methods.